client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue

```diff
@@ -55,6 +56,7 @@
         needs-hidden-input
         required
         data-dp-validate-if="#videoTitle!=='', #videoDescription!==''"
+        data-cy="uploadVideo"
```

Comment:
```suggestion
        data-cy="customerSettings:videoUpload"
```

And in line 66, the textarea needs a data-cy, too. I'd propose `customerSettings:videoDescription` as a value for it.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3632/commits/b41db82d5cc9234745a51ba2dc66fce5d9a2dede
Thnaks!

Reply:
Are you still planning to use the above suggestion?

Reply:
Yes, done
thanks